packages/js/product-editor/src/components/editor/editor.tsx

```diff
@@ -47,7 +48,12 @@ export function Editor( { product, settings }: EditorProps ) {
 					<FullscreenMode isActive={ false } />
 					<SlotFillProvider>
 						<InterfaceSkeleton
-							header={ <Header title={ product.name } /> }
+							header={
+								<Header
+									productId={ product.id }
+									title={ product.name }
```

Comment:
Just a note here that when the `product` is new the `name` doesn't get updated.
I assume this because we are not passing in the the product from a `useSelect` hook in the `product-page` but we are for existing products.
I think we probably don't want to do that anyway, and only pass the product in a constant and for the rest rely on the `selectors` within.